src/settings.ts

```diff
@@ -551,6 +551,12 @@ export const ALL_BURN_TEMPLATES: Array<VenueTemplate> = [
   VenueTemplate.themecamp,
 ];
 
+// @debt Refactor this constant into types/templates along with the rest *_TEMPLATES constants
```

Comment:
This array is different to the situation where the other venue related things talk about being moved to `types/venue`. This is a configurable setting, and belongs here. Most of the others are related to inherent properties of the venue types, not things that can be configured.

Please remove the `@debt` comment from this line.

Reply:
Actually after the fix in naming as called out in https://github.com/sparkletown/sparkle/pull/1557/commits/d404b6c3244edd64977f6e9cec73f64506bb5fca#r655090559 this `@debt` comment may be relevant again.

Reply:
kept the comment b4c9d461ba2e3bb154e34624998df49e1c40bf73